proto/greenfield/virtualgroup/events.proto

```diff
@@ -90,12 +105,10 @@ message EventUpdateLocalVirtualGroup {
     (gogoproto.customtype) = "Uint",
     (gogoproto.nullable) = false
   ];
-  // The src_global_virtual_group_id is gvgid of lvg before migrate
-  uint32 src_global_virtual_group_id = 3;
-  // The dst_global_virtual_group_id is gvgid of lvg after migrate
-  uint32 dst_global_virtual_group_id = 4;
+  // The global_virtual_group_id is gvgid of lvg after migrate
+  uint32 global_virtual_group_id = 3;
   // The stored size of all the objects stores in this lvg
-  uint64 stored_size = 5;
+  uint64 stored_size = 4;
 }
```

Comment:
uint64 total_charge_size = 5;
  do we need add this one when we update lvg

Reply:
We need discuss with @forcodedancing later.